packages/flutter/lib/src/rendering/table.dart

```diff
@@ -347,7 +347,10 @@ enum TableCellVerticalAlignment {
 
   /// Cells with this alignment are sized to be as tall as the row, then made to fit the row.
   /// If all the cells have this alignment, then the row will have zero height.
-  fill
+  fill,
+
+  /// Cells with this alignment are sized to be the same height as the tallest cell in the row.
+  intrisicHeight
```

Comment:
Just noticed this typo here - can you update everywhere?

```suggestion
  intrinsicHeight
```

Reply:
Yes! I'll do it tomorrow morning. Thanks for the comment. 

Intrinsic isn't an easy word to write 😆